src/main/java/com/denizenscript/ddiscordbot/DiscordCommand.java

```diff
@@ -45,9 +56,33 @@
     // Use to message a Discord channel.
     // - discord id:mybot message channel:<discord[mybot].server[Denizen].channel[bot-spam]> "Hello world!"
 
+    // @Usage
+    // Use to add a role on a user in a Discord guild.
+    // - discord id:mybot role add user:<def[user_id]> role:324782895 guild:<def[guild_id]>

```

Comment:
What are these numbers? If there just samples of role id numbers, should document as `<def[role_id]>` or similar, to fit with the other docs